src/pages/index.mustache

```diff
@@ -111,8 +111,8 @@
 
 						<!-- Recommended Section -->
 						<div class="plugin-section main-section">
-							<div class="highlighted-heading heading -with-sort-option">
-								<h3 id="recommended-section">Recommended plugins</h3>
+							<div class="highlighted-heading heading-with-sort-option">
```

Comment:
```suggestion
							<div class="highlighted-heading heading -with-sort-option">
```

Please keep the space — [see the RCSS documentation on variants](https://ricostacruz.com/rscss/variants.html).